### PR TITLE
flask-cors integration

### PR DESCRIPTION
**Pull request by mfranco**
@Rondineli @niallryan

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 setup(
     name='Flask-Philo',
-    version='3.6.2',
+    version='3.6.7',
```
Comment (niallryan) [inline setup.py]: Looks good, but maybe this should be `3.7.0`

**Comment by Rondineli** [comment]
:+1:

File: setup.py (hunk 1 above)
Comment (mfranco) [reply setup.py]: done :+1:

**Comment by mfranco** [comment]
:+1:

**Comment by cormacp** [comment]
👍

**Pull request merged**